charter.md

```diff
@@ -1,10 +1,10 @@
-[RFC 5033](https://datatracker.ietf.org/doc/rfc5033/) describes a Best Current Practice to evaluate new congestion control algorithms as Experimental or Proposed Standard RFCs. TCP was the dominant consumer of this work, and proposals were typically discussed in research groups, for example the Internet Congestion Control Research Group (ICCRG).
+Congestion control is the process a network sender uses to determine the rate at which to send data. A congestion control algorithm balances sending sufficient data to make meaningful progress for a user with avoiding filling buffers in the network and overloading the network path. Better congestion control algorithms can help improve application performance, especially latency under load, while safeguarding overall network performance for all senders.
```

Comment:
It would be nice to also briefly say why congestion control standardization is valuable, since it is not generally an interoperability concern.

Reply:
I love that latency under load is mentioned, but don't think "especially" is a good choice of words. "such as", or "for example" might be better?

Reply:
```suggestion
Congestion control is the process a network sender uses to determine the rate at which to send data. A congestion control algorithm balances sending sufficient data to make meaningful progress for a user with avoiding filling buffers in the network and overloading the network path. Better congestion control algorithms can help improve application performance, such as latency under load, while safeguarding overall network performance for all senders.
```
Getting back to this PR, "such as" sounds good to me

Reply:
> It would be nice to also briefly say why congestion control standardization is valuable, since it is not generally an interoperability concern.

Agreed. We could reuse text from the Introduction of 5033bis here:

> […] a specification for a congestion control algorithm provides a number of advantages:
>
> It can help implementers, operators, and other interested parties develop a shared understanding of how the algorithm works and how it is expected to behave in various scenarios and configurations.
>
> It can help potential contributors understand the algorithm, which can make it easier for them to suggest improvements and/or identify limitations. Furthermore, the specification can help multiple contributors align on a consensus change to the algorithm.
> 
> A specification that is accessible to anyone can circumvent the issue that some implementers may be unable to read open source reference implementations due to the constraints of some open source licenses.

What does everyone else think?

Reply:
Updated in [46b0462](https://github.com/ietf-wg-ccwg/wg-materials/pull/2/commits/46b04624062cfcef62859e3f65895d5d7ebc57b8)!